### ISSUE TITLE
Exception in AnalyzeInvocationExpressionSyntax

### ISSUE THREAD

**Issue by jnyrup** [KThaulow/CodeAnalyzer#19]
`TimeFormatAnalyzer`
```
Severity	Code	Description	Project	File	Line	Suppression State	Detail Description
Warning	AD0001	Analyzer 'CodeAnalyzer.Analyzers.TimeFormatAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.	ConsoleApp44		1	Active	Analyzer 'CodeAnalyzer.Analyzers.TimeFormatAnalyzer' threw the following exception:
'Exception occurred with following context:
Compilation: ConsoleApp44
SyntaxTree: C:\Users\jonas\source\repos\ConsoleApp44\ConsoleApp44\Program.cs
SyntaxNode: F(Date) [InvocationExpressionSyntax]@[505..512) (18,12)-(18,19)

System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeAnalyzer.Analyzers.TimeFormatAnalyzer.AnalyzeInvocationExpressionSyntax(SyntaxNodeAnalysisContext context, INamedTypeSymbol dateTimeSymbol)
   at CodeAnalyzer.Analyzers.TimeFormatAnalyzer.<>c__DisplayClass8_0.<Initialize>b__1(SyntaxNodeAnalysisContext nodeContext)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__44`1.<ExecuteSyntaxNodeAction>b__44_0(ValueTuple`2 data)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info)
-----
'.
```

`DoubleParseAnalyzer`
```
Severity	Code	Description	Project	File	Line	Suppression State	Detail Description
Warning	AD0001	Analyzer 'CodeAnalyzer.Analyzers.DoubleParseAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.	ConsoleApp44		1	Active	Analyzer 'CodeAnalyzer.Analyzers.DoubleParseAnalyzer' threw the following exception:
'Exception occurred with following context:
Compilation: ConsoleApp44
SyntaxTree: C:\Users\jonas\source\repos\ConsoleApp44\ConsoleApp44\Program.cs
SyntaxNode: F(Date) [InvocationExpressionSyntax]@[505..512) (18,12)-(18,19)

System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeAnalyzer.Analyzers.DoubleParseAnalyzer.AnalyzeInvocationExpressionSyntax(SyntaxNodeAnalysisContext context, INamedTypeSymbol doubleSymbol)
   at CodeAnalyzer.Analyzers.DoubleParseAnalyzer.<>c__DisplayClass8_0.<Initialize>b__1(SyntaxNodeAnalysisContext nodeContext)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__44`1.<ExecuteSyntaxNodeAction>b__44_0(ValueTuple`2 data)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info)
-----
'.
```

`DateTimeStringAnalyzer`
```
Severity	Code	Description	Project	File	Line	Suppression State	Detail Description
Warning	AD0001	Analyzer 'CodeAnalyzer.Analyzers.DateTimeStringAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'.	ConsoleApp44		1	Active	Analyzer 'CodeAnalyzer.Analyzers.DateTimeStringAnalyzer' threw the following exception:
'Exception occurred with following context:
Compilation: ConsoleApp44
SyntaxTree: C:\Users\jonas\source\repos\ConsoleApp44\ConsoleApp44\Program.cs
SyntaxNode: F(Date) [InvocationExpressionSyntax]@[505..512) (18,12)-(18,19)

System.NullReferenceException: Object reference not set to an instance of an object.
   at CodeAnalyzer.Analyzers.DateTimeStringAnalyzer.AnalyzeInvocationExpressionSyntax(SyntaxNodeAnalysisContext context, INamedTypeSymbol doubleSymbol)
   at CodeAnalyzer.Analyzers.DateTimeStringAnalyzer.<>c__DisplayClass8_0.<Initialize>b__1(SyntaxNodeAnalysisContext nodeContext)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__44`1.<ExecuteSyntaxNodeAction>b__44_0(ValueTuple`2 data)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info)
-----
'.
```

**Comment by jnyrup** [comment]
Try this:

```c#
namespace ConsoleApp45
{
    class Program
    {
        static void Main(string[] args)
        {
            var str = nameof(args);
        }
    }
}
```

**Comment by KThaulow** [comment]
Fixed in commit 66ca76d6b7acc3d320e488590eb034f8348c5d9f
